### ISSUE TITLE
`new PCMFormat` - object is not a function

### ISSUE THREAD

**Issue by dy** [raymond-h/pcm-format#1]
Hello! I try to use the module in node:

``` js
var PCMFormat = require('pcm-format');
new PCMFormat({float: true}, {float: false});
```

And get that error:

``` sh
c:\Users\dmitry\Dropbox\Projects\audio-generator\test.js:11
var pcmFormat = new PCMFormat(
                ^
TypeError: object is not a function
    at Object.<anonymous> (c:\Users\dmitry\Dropbox\Projects\audio-generator\test.js:11:17)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Function.Module.runMain (module.js:497:10)
    at startup (node.js:119:16)
    at node.js:935:3
npm ERR! Test failed.  See above for more details.
npm ERR! not ok code 0
```

Node version is 0.10.40.

That seems to happen because of babel incompatible commonJS transform - it exports `default` as an object property.

**Comment by raymond-h** [comment]
Indeed. However, apparently if you only export a default value and nothing else, the exported value is also set onto `module.exports`. I'll do that.

**Comment by raymond-h** [comment]
Fixed in ed0ef19dc9ff8c5f1375532c8720dd02fdad37d3.

**Comment by dy** [comment]
Wow, blazingly fast. Thank you!
